cms/static/js/views/feedback.js

```diff
@@ -1,146 +0,0 @@
-define(["jquery", "underscore", "underscore.string", "backbone", "js/utils/templates"],
```

Comment:
Can you not delete these files but instead move them from cms to common. I don't want to lose the history.